tests/integration/suite/sentry/utils.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metadata
+
+import (
+	"crypto"
+	"crypto/rand"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"fmt"
+	"time"
+
+	"github.com/lestrrat-go/jwx/v2/jwa"
+	"github.com/lestrrat-go/jwx/v2/jwk"
+	"github.com/lestrrat-go/jwx/v2/jwt"
+)
+
+const (
+	// Trust domain for Sentry
+	sentryTrustDomain = "localhost"
+	// Namespace for sentry
+	sentryNamespace = "default"
+)
+
+// Keys used to sign and verify JWTs
+const (
+	jwtSigningKeyPubJSON  = `{"kid":"mykey","kty":"EC","alg":"ES256","crv":"P-256","x":"UMn1c2ioMNi2DqvC8hdBVUERFZ97eVFsNVcQIgR0Hso","y":"uT1a0P3UOLiObve2-pOMFx2BVzLz5rFtU-qmQBPWwd0"}`
+	jwtSigningKeyPrivJSON = `{"kid":"mykey","kty":"EC","alg":"ES256","crv":"P-256","d":"5wV7hDpqt1L3uaXa1Xj7X3ieaV9A-Hyj2Kv-qxpwSjM","x":"UMn1c2ioMNi2DqvC8hdBVUERFZ97eVFsNVcQIgR0Hso","y":"uT1a0P3UOLiObve2-pOMFx2BVzLz5rFtU-qmQBPWwd0"}`
```

Comment:
There is no need for these to be hardcoded. As a rule of thumb, never commit private key material. Please remove and generate the key at test runtime.

Reply:
The reason why I hardcoded these is because generating a private key can take time and the ITs need to run quickly.

Although I guess that generating a P-256 key should be relatively fast unlike RSA keys. I'll make the change and hope it doesn't cause issues :)